scripts/install/prowlarr.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+# *Arr Installer for Prowlarr
+# Refactored from existing files by FlyingSausages and others
+# Bakerboy448 2021 for Swizzin
+
+#shellcheck source=sources/functions/utils
+. /etc/swizzin/sources/functions/utils
+
+app_name="prowlarr"
+if [ -z "$PROWLARR_OWNER" ]; then
+    if ! PROWLARR_OWNER="$(swizdb get "$app_name/owner")"; then
+        PROWLARR_OWNER="$(_get_master_username)"
+        echo_info "Setting ${app_name^} owner = $PROWLARR_OWNER"
+        swizdb set "$app_name/owner" "$PROWLARR_OWNER"
+    fi
+else
+    echo_info "Setting ${app_name^} owner = $PROWLARR_OWNER"
+    swizdb set "$app_name/owner" "$PROWLARR_OWNER"
+fi
+user="$PROWLARR_OWNER"
+app_configdir="/home/$user/.config/${app_name^}"
+app_group="$user"
+app_port="9696"
+app_reqs=("curl" "sqlite3")
+app_servicefile="$app_name".service
+app_dir="/opt/${app_name^}"
+app_binary="${app_name^}"
+app_lockname="$app_name"
```

Comment:
This should remove any dashes if they exist in `$app_name`
```suggestion
app_lockname="${app_name//-/}"
```